qt/applications/workbench/workbench/projectrecovery/recoverygui/recoveryfailureview.py

```diff
@@ -62,7 +62,7 @@ def change_start_mantid_button(self, string):
 
     def closeEvent(self, _):
         if self.editor is not None:
-            self.editor.disconnect_editor_from_sig_process()
+            self.editor.disconnect_to_progress_reports()
```

Comment:
Just the pedant in me: **\_from\_** instead of **\_to\_**?

Reply:
Yep good spot :+1: